faq.md

```diff
@@ -13,6 +21,10 @@ Yes, see [Building](building) for more details.
 
 Well that's great to hear! Please open an issue so that I can keep track of your request. I'll gladly implement what you've asked for, as long as it's feasible.
 
+### Are shaders supported on the 3DS?
+
+No. The 3DS doesn't support OpenGL Shader Language, which is what LÖVE uses. Shader programs for PICA200 (the 3DS GPU) need to be written in assembly and pre-compiled.
```

Comment:
Should specifically say it's [picasso](https://github.com/devkitPro/picasso) shader code (which is assembled to PICA200 code).